cms/djangoapps/contentstore/views/item.py

```diff
@@ -457,6 +459,10 @@ def _save_xblock(user, xblock, data=None, children_strings=None, metadata=None,
         else:
             data = old_content['data'] if 'data' in old_content else None
 
+        if fields:
```

Comment:
Ah, I see. Is the plan to change other "fields" elements that are explicitly defined into "fields"?